src/utils/auth.ts

```diff
@@ -189,7 +186,7 @@ export const listenForAuthorizationCode = async ({
  * @throws if the token request fails
  */
 export const requestForIdToken = async (body: URLSearchParams) => {
-  const url = `https://${AUTH0ֹֹֹֹֹ_VANITY_URI}/oauth/token`;
+  const url = `${AUTH0ֹֹֹֹֹ_ISSUER}/oauth/token`;
```

Comment:
i like that the `https://` is no longer mandatory

Reply:
me too. this came from a misunderstanding on my part with how auth0 worked